examples/api/test/widgets/navigator/navigator.restorable_push_and_remove_until.0_test.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/navigator/navigator.restorable_push_and_remove_until.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('It pushes and restorable route and pops it', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('It pushes a restorable route and pops it', (WidgetTester tester) async {
```

Not sure if 'a' was the expected word?

Reply:
Arf! Sorry for that 😞 I fixed it in [test: Fix test descriptions](https://github.com/flutter/flutter/pull/157487/commits/1a4e9994e4def91b0033e2e8b3e4ff6dee137a93)